plugins/Terminal/js/constants/helper.js

```diff
@@ -0,0 +1,5 @@
+export const REGULAR_COMMAND = -1
+export const WALLET_SEED = 0
+export const WALLET_UNLOCK = 1
+export const HELP = 2
+export const HELP_QMARK = 3
```

Comment:
not really what I had in mind... the values returned by `commandType` are still "magic."

But if you revisit this code in a month, I'm sure you'll be brimming with ways to improve it, so lets not sweat the small stuff for now.